src/app/pages/notifications/notifications.page.ts

```diff
@@ -31,6 +32,8 @@ export class NotificationsPage extends NotificationBase implements OnInit {
 
   unsupportedPrysm: boolean = false
 
+  noGoogle = false
```

Comment:
Since this caches the correct value as soon as the constructor is called, it could be used wherever `isNoGoogleFlavor()` is used.

Reply:
You'll find it being used in the template for this very reason :)

Reply:
As discussed, won't be changed. The constructor works async and forcing it to wait here in order to have the saved value would delay the boot time.